src/stc/PlatWX.cpp

```diff
@@ -3290,6 +3301,19 @@ void ListBoxImpl::Create(Window &parent, int WXUNUSED(ctrlID),
                          bool WXUNUSED(unicodeMode_), int technology_) {
     wid = new wxSTCListBoxWin(GETWIN(parent.GetID()), &m_listBox, m_visualData,
                               lineHeight_, technology_);
+
+    if ( m_listBoxWidth >= -1 )
```

Comment:
Just checking: is this really supposed to be `>= -1` and not `> -1` (or maybe `!= -1`)?

Reply:
Whoops, good catch. I meant that to be strict so either `> - 1` or `>= 0` so that it only sets the width in the underlying control if it has been modified from the default.

Reply:
OK, I've changed this to use `>= 0` and will merge it, thanks!